homeassistant/components/deconz/sensor.py

```diff
@@ -205,6 +208,17 @@ class DeconzSensorDescription(SensorEntityDescription, DeconzSensorDescriptionMi
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=LIGHT_LUX,
     ),
+    DeconzSensorDescription[Moisture](
+        key="moisture",
+        supported_fn=lambda device: device.moisture is not None,
+        update_key="moisture",
+        value_fn=lambda device: round(device.moisture / 100, 1),
```

Comment:
You should be able to remove the rounding

Reply:
Yes, I am waiting for https://github.com/Xyaren/deconz/blob/master/pydeconz/models/sensor/moisture.py#L41 which would remove the scaling from the integration and move it to the lib.

Reply:
I removed the rounding.